README.md

```diff
@@ -1 +1,35 @@
-# Todo-list-peer-to-peer
\ No newline at end of file
+# ToDoList
+
+A  ToDo list with JavaScript that helps users write down list of activities they want to accomoplish within a specicific period of time.
+
+# Build with 
+ - HTML 
+ - CSS
+ - JavaScript
+
+👤 **Author**
+- GitHub: [@githubhandle](https://github.com/SirriRyisa)
+- Twitter: [@twitterhandle](https://twitter.com/N_Ryisa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ryisa-sirri-ngwa-a30013202)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/SirriRyisa/ToDoList/issues).
+
+# Run Project
+
+Copy from github repo and clone into your PC
+
+Run npm install to install all dependencies 
+
+npm start to run the application on a browser
+
+# Project live Demo
+
+- link [LiveDemon]( https://sirriryisa.github.io/ToDoList/)
```

Comment:
- [ ] Optional

Great Professional README. Please consider correcting grammar in Demo description text, to stand as a careful and great developer with an eye for detail.